src/getters/getConfigFromChain.ts

```diff
@@ -0,0 +1,32 @@
+import { Contract } from 'ethers'
+import { StreamrConfig, streamrConfigABI } from '@streamr/network-contracts'
+import { getSigner } from '~/shared/stores/wallet'
+import { BNish } from '~/utils/bn'
+import { errorToast } from '~/utils/toast'
+import { defaultChainConfig } from './getChainConfig'
+
+/**
+ * When we will be needing more of the fields, simply expand the return type and add them
+ */
+export const getConfigFromChain = async (): Promise<{
+    maxPenaltyPeriodSeconds: BNish
+    minimumStakeWei: BNish
+} | null> => {
+    const signer = await getSigner()
```

Comment:
Here, the other point where this utility can explode but is outside of that try…catch below.